l1-contracts/test/helpers/UnitTestHelper.sol

```diff
@@ -72,14 +78,18 @@ contract UnitTestHelper is Test, BaseScript {
         mockEigenPodManager = new MockEigenPodManager();
         mockDelegationManager = new MockDelegationManager();
         mockAVSDirectory = new MockAVSDirectory();
-
+        mockStrategyManager = new MockStrategyManager();
+        mockRewardsCoordinator = new MockRewardsCoordinator(IStrategyManager(address(mockStrategyManager)));
         AVSDeployment memory avsDeployment = new DeployEverything().run(
```

Comment:
Can we fix our old mistakes and use latest best practices and fix things on the go to use named parameters(or whatever it is called) for calls that use the same types(address,address,address...)
```
new DeployEverything().run({
    eigenPodManager: address(mockEigenPodManager),
    eigenDelegationManager: address(mockDelegationManager),
    ...
    ...
});